.github/workflows/swift-toolchain.yml

```
@@ -2118,9 +2118,118 @@ jobs:
           symbolsFolder: ${{ github.workspace }}/BinaryCache
           searchPattern: '**/*.exe'
 
+  debugging_tools:
+    name: Debugging Tools
+    environment: "kendal/testing"
+    needs: [context, compilers, devtools, sdk]
+    runs-on: ${{ needs.context.outputs.windows_build_runner }}
+    env:
+      TOOLCHAIN_PATH: ${{ github.workspace }}/BuildRoot/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain/usr/bin
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - arch: amd64
+            cpu: x86_64
+            triple: x86_64-unknown-windows-msvc
+
+          # TODO(125): Fix the arm64 build and uncomment this.
```

Please qualify the issue

```suggestion
          # TODO(thebrowsercompany/swift-build#125): Fix the arm64 build and uncomment this.
```